transformations/replace_abbreviation_and_acronyms/abbreviations.txt

```diff
@@ -0,0 +1,130 @@
+ACCT:Account
```

Comment:
You might want to add a Data and Code provenance section in the README which mentions the source of this data.